sources/frontend.c

```diff
@@ -743,17 +743,10 @@ od_frontend_remote(od_client_t *client)
 	od_server_t *server;
 	od_instance_t *instance = client->global->instance;
 	for (;;) {
-		/* we go to the eternal wait
```

Comment:
is this change related to PR topic?

Reply:
this is completely wrong and should be removed,  test on this behaviour is wrong too